### DISCUSSION TITLE
Configurable column_encryption cipher

### DISCUSSION THREAD

**Discussion by baelter** [jeremyevans/sequel#2215] [General]
Are you open to making the cipher used in `column_encryption` configurable?
`aes-256-gcm` is not supported by `pgcrypto` which is a bit inconvenient if you want to decrypt outside of Sequel.

I can give it a go if you think it's a good idea @jeremyevans

**Comment by jeremyevans** [comment]
I don't plan on making the cipher configurable unless there are cryptographic issues discovered in `aes-256-gcm`.  So if you want to use a different cipher, I recommend you copy the current `column_encryption` plugin and modify it.
